src/parser/transform/statement/transform_alter_sequence.cpp

```diff
@@ -1,9 +1,9 @@
-#include "duckdb/parser/transformer.hpp"
-#include "duckdb/parser/statement/alter_statement.hpp"
-#include "duckdb/parser/parsed_data/alter_table_info.hpp"
-#include "duckdb/parser/parsed_data/create_sequence_info.hpp"
 #include "duckdb/common/enum_class_hash.hpp"
 #include "duckdb/common/unordered_set.hpp"
+#include "duckdb/parser/parsed_data/alter_table_info.hpp"
+#include "duckdb/parser/parsed_data/create_sequence_info.hpp"
+#include "duckdb/parser/statement/alter_statement.hpp"
+#include "duckdb/parser/transformer.hpp"
```

Comment:
Just curious, did all this rearranging of includes happen because of some tidy tool? To make them alphabetically?

Reply:
Yeah, it was a vscode thing, I can probably undo it.

Reply:
No all good, I think it is neat. Was just wondering about it.